src/http/common.cr

```diff
@@ -38,10 +38,7 @@ module HTTP
         if body_type.prohibited?
           body = nil
```

Comment:
Would it make sense to start the `when EndOfRequest` block as 
```
body = 
  if body_type.prohibited?
 [...]
```

Reply:
No, it'd make the diff larger for no practical benefit.

It's style, but many people prefer to assign in each if branch instead of use if as the right hand side of an expression.